src/utils/featheryClient/index.ts

```diff
@@ -474,6 +528,23 @@ export default class FeatheryClient extends IntegrationClient {
     );
   }
 
+  async submitCustom(customKeyValues: { [key: string]: any }, override = true) {
```

Comment:
if override is false, we should flush it immediately. override=false and override=true calls have different behavior (they either override exisitng field data on the backend or not) and can't be combined. Alos override=false is only used in one place that doesn't need to be debounced so it's fine to trigger immediately